style.css

```diff
@@ -0,0 +1,94 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+a:hover {
+  color: #172b4d;
+}
+
+.html.body {
+  font-family: Poppins, sans-serif;
+}
+
+.blue {
+  color: #6070ff;
+}
+
+.header_nav {
+  display: flex;
+  min-height: 10vh;
+  padding: 10px 20px;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.header_logo {
+  font-size: 22px;
+  font-weight: 700;
+  line-height: 20px;
+}
+
+.header_logo a {
+  text-decoration: none;
+}
+
+.header_menu {
+  padding: 3px 5px;
+}
+
+.menu_icon {
+  width: 24px;
+  height: auto;
+}
+
+.headline {
+  padding: 25px;
+  display: flex;
+  min-height: 50vh;
+  background-size: contain;
+  flex-direction: column;
+  background-repeat: space;
+  background-image: url("./images/header-shapes mobile.png");
+}
```

Comment:
[OPTIONAL] 👍🏽 
- I think it will be better to use a `background-repeat: no-repeat;` instead as this will stop the repetition of your background image across your portfolio. I think you may have downloaded a background image with a width of less than 375px, so it doesn't properly cover the entire mobile screen. 

I guess you can look into it when you have the time. If I get a solution for you before then, I will reach out to you as well. 🍷 🌸

Reply:
sure thing, Thanks